standalone-metastore/metastore-server/src/main/sql/hive/upgrade-4.0.0-to-4.1.0.hive.sql

```diff
@@ -226,5 +226,96 @@ TBLPROPERTIES (
 
 CREATE OR REPLACE VIEW `VERSION` AS SELECT 1 AS `VER_ID`, '4.1.0' AS `SCHEMA_VERSION`,
   'Hive release version 4.1.0' AS `VERSION_COMMENT`;
+  
+CREATE OR REPLACE VIEW `COMPACTIONS`
```

Comment:
@difin  same should be done for both SYS and INFORMATION_SCHEMA

Reply:
Do you mean changes in derby/mssql/mysql/oracle/postgres upgrade scrips?
This change is only in `COMPACTIONS` view, it exists only in Hive catalog.

Reply:
Done